### ISSUE TITLE
[request] getY /get X

### ISSUE THREAD

**Issue by newboerg** [fesch/Structorizer.Desktop#424]
Would it be possible to add the following Turtle functions in the next release?

getX()
getY()

As we already have gotoX/Y this would add a lot more possibilities for the turtle programs

**Comment by codemanyak** [comment]
Technically this is not of course a problem. And a teacher may still prohibit their use if she/he wants to force the students to track the turtle movements themselves by means of the algorithm.
So this is a yes.

**Comment by codemanyak** [comment]
Should we add getOrientation() to the enhancement request on this occasion?

**Comment by newboerg** [comment]
yes, this would be convenient.

**Comment by codemanyak** [comment]
I wrote:
> Technically this is not of course a problem.

Well, this was perhaps a little euphemistic. It's more complicated than I thought in the first moment (and I guess that's the reason it hasn't been implemented by now): Turtleizer had only offered procedures before. Their call forms an entire instruction line. With function calls it's different: They may be nested arbitrarily deeply, such that Turtleizer **functions** must first be picked out of the instruction line and be computed in advance. I'm working on a solution.

**Comment by newboerg** [comment]
Maybe a little workaround Like with recursive function calls. Which cannot bei nestet i think?

**Comment by codemanyak** [comment]
@newboerg 
Thanks for trying to simplify my work, but there's no need: A prototype is already working.

**Comment by GitMensch** [comment]
This item can be closed already, couldn't it?.

**Comment by codemanyak** [comment]
@GitMensch 
For the requestor, it hasn't been solved while the new version isn't shipped. The item is prepared in a way that it will automatically be closed on merging the fix to the (fesch) master branch on delivering.

**Comment by GitMensch** [comment]
@codemanyak thank you for the information, I wasn't aware of the tracking for this, but now see the commit log show "fixes" + issue-number which is recognized by GitHub and GitHub showing "This issue will close once commit ... is merged into the 'master' branch".
